### PR TITLE
Add 10 faculty members from Shenzhen University to CSrankings

### PR DESCRIPTION
**Pull request by zmh217**
# Contributing to CSrankings

Thanks for contributing to CSrankings! Please read and indicate you agree with **all** these guidelines to getting your pull request accepted. Note that pull requests may take some time to get merged (please don't contact us for at least three months for updates, unless you are a sponsor - see below).

**If you find CSrankings useful, [please consider becoming a sponsor](https://github.com/sponsors/csrankings).**

**NOTE:** IF YOU DO NOT FOLLOW THE STEPS BELOW, YOUR PULL REQUEST WILL BE SUMMARILY REJECTED.
You must read and check **all** the boxes below by filling them in with an X or your PR will be rejected.

**The Basics**

- [X] All pull requests and issues must come from non-anonymous accounts. Make sure your GitHub profile contains your full name.

- [X] Use a reasonable title that explains what the PR corresponds to (as in, not "Update csrankings-x.csv").

- [X] Combine multiple updates to a single institution into a **single PR.**

- [X] Only submit one pull request per institution.

- [X] Do not modify any files except `csrankings-[a-z].csv` or (if needed) `country-info.csv` and `old/industry.csv` (see below).

- [X] Do not use Excel to edit any .csv files; Excel incorrectly tries to
convert some Google Scholar entries to formulas, corrupting the
database. Use the GitHub user interface or a text editor like emacs or NotePad instead.

- [X] Insert new faculty **in alphabetical order** (not at the end) in the appropriate `csrankings-[a-z].csv` files. **Do not modify `csrankings.csv`, which is auto-generated.**

- [X] Check to make sure that you have no spaces after commas, or any missing fields.

- [X] Check to make sure the home page is correct.

- [X] Make sure the Google Scholar IDs are just the alphanumeric identifier (not a URL or with `&hl=en`).

- [X] Check to make sure the name corresponds to the DBLP entry (look it up at http://dblp.org).

- [X] If a faculty member is not in a CS department or similar, include a comment explaining how they meet the inclusion criteria (see below).

**Inclusion criteria**

- [X] Make sure that any faculty you add meet the inclusion
criteria. Eligible faculty include only full-time, tenure-track research
faculty members on a given campus who can *solely* advise PhD students in
Computer Science. Faculty not in a CS department or similar who can
advise PhD students in CS can be included regardless of their home
department. **Provide justification, pointing to specific home pages
showing how faculty not in a CS department meet the inclusion criteria,
e.g. showing a courtesy appointment in CS.** Faculty must also have a 75%+ time appointment (check
`old/industry.csv` for faculty who are now more than 25% in industry).

**Updating an affiliation or home page**

- [X] Update affiliations, home pages, and Google Scholar entries by modifying `csrankings-[a-z].csv`. For the Google Scholar entry, just use the alphanumeric identifier in the middle of the URL. If none is there, put `NOSCHOLARPAGE`.

**Adding one or more faculty members (including an entire department)**

- [X] If the department is not yet listed in CSrankings, the entire CS faculty needs to be added (not just one faculty member).

- [X] Enter each faculty member's [DBLP](http://dblp.org) name, home page, and Google Scholar entry (just the alphanumeric identifier, not the whole URL) by modifying `csrankings-[a-z].csv` (**the letters correspond to the first letter of the faculty members' names**); include disambiguation suffixes like `0001` as needed. If the faculty entry is currently ambiguous, please do not include them. Send mail to the DBLP maintainers (dblp@dagstuhl.de) with a few publications by a particular faculty member; also, open an issue so that when the DBLP database is updated, that faculty member's information can be added.

- [X] If DBLP has multiple entries for this person, *all of them need to be listed*. Do not update `dblp-aliases.csv`.

- [X] If the institution you are adding is not in the US,
update `country-info.csv` and add *all* of the faculty in the CS department.

**(Advanced) Quick contribution via a shallow clone** 

We recommend that you use the GitHub web user interface to make changes. However, it may be more convenient
to clone the repository for larger-scale changes.

However, a full clone of the CSrankings repository is almost 2GB.  To
contribute a change without creating a full local clone of the
CSrankings repo, you can perform a _shallow clone_.  To do so, follow these
steps:

1. Fork the CSrankings repo.  If you have an existing fork, but it is not up to date with the main repository, this technique may not work.  If necessary, delete and re-create your fork to get it up to date.  (Do **not** delete your existing fork if it has unmerged changes you want to preserve!)

1. Do a shallow clone of your fork: `git clone --depth 1 https://github.com/yourusername/CSrankings`.  This will only download the most recent commit, not the full git history.

1. Make your changes on a branch, push them to your clone, and create a pull request on GitHub as usual.

If you want to make another contribution and some time has passed, perform these steps again, creating a fresh fork and shallow clone.

### CODE REVIEW

File: csrankings-j.csv [hunk 1]
```diff
@@ -967,6 +967,7 @@ Jiajing Zhu,UESTC,https://sise.uestc.edu.cn/info/1036/11717.htm,NOSCHOLARPAGE
 Jiajun Bu,Zhejiang University,http://mypage.zju.edu.cn/bjj,OgZP2okAAAAJ
 Jiajun Chen 0001,Nanjing University,https://cs.nju.edu.cn/chenjiajun/index_en.htm,WIF7VaoAAAAJ
 Jiajun Liu,Renmin University of China,https://sites.google.com/site/drjiajunliu/homepage,xZGKQkcAAAAJ
+Jiajun Wen 0001,Shenzhen University,https://csse.szu.edu.cn/en/pages/user/index?id=835,VmiSYTgAAAAJ
```
Comment (emeryberger) [inline csrankings-j.csv:970]: Senior Research Scientist, not eligible for inclusion.

File: csrankings-s.csv [hunk 2]
```diff
@@ -1200,6 +1200,7 @@ Shriram Krishnamurthi,Brown University,https://cs.brown.edu/~sk,Cxwj5hEAAAAJ
 Shrisha Rao 0001,IIIT Bangalore,http://www.iiitb.ac.in/faculty_page.php?name=ShrishaRao,SxYeZQcAAAAJ
 Shruti Vyas,University of Central Florida,https://www.cs.ucf.edu/person/shruti-vyas/,15YqUQUAAAAJ
 Shu Kong,University of Macau,https://www.fst.um.edu.mo/people/skong/,sm9FdLoAAAAJ
+Shu Yang 0002,Shenzhen University,https://ieeexplore.ieee.org/author/37086450194,JKR65R0AAAAJ
```
Comment (emeryberger) [inline csrankings-s.csv]: Associate Researcher, not TT faculty. Remove.

File: csrankings-t.csv [hunk 3]
```diff
@@ -155,6 +155,7 @@ Tanzima Z. Islam,Texas State University,https://www.tanzimaislam.com,q56uneAAAAA
 Tanzima Zerin Islam,Texas State University,https://www.tanzimaislam.com,q56uneAAAAAJ
 Tao A. Xiang,University of Surrey,http://www.eecs.qmul.ac.uk/~txiang,MeS5d4gAAAAJ
 Tao Chen 0001,Loughborough University,https://www.lboro.ac.uk/departments/compsci/staff/academic-teaching/tao-chen,K4teyvoAAAAJ
+Tao Dai 0001,Shenzhen University,https://openreview.net/profile?id=~Tao_Dai3,MqJNdaAAAAAJ
```
Comment (emeryberger) [inline csrankings-t.csv]: Replace openreview with actual web page.

File: csrankings-x.csv [hunk 4]
```diff
@@ -444,6 +444,7 @@ Xu Liu 0001,North Carolina State University,https://xl10.github.io,qhLT67EAAAAJ
 Xu Liu 0006,Xidian University,https://web.xidian.edu.cn/xuliu/,_09bkMgAAAAJ
 Xu Sun 0001,Peking University,https://xusun.org,NOSCHOLARPAGE
 Xu Tang,Xidian University,https://sites.google.com/site/xutanghomepage/,lmskeBMAAAAJ
+Xu Wang 0006,Shenzhen University,https://openreview.net/profile?id=~Xu_Wang17,XNDHhIEAAAAJ
```
Comment (emeryberger) [inline csrankings-x.csv]: Replace openreview with actual home page

**Review by emeryberger — CHANGES_REQUESTED** [review]

File: csrankings-j.csv (hunk 1 above)
Comment (zmh217) [reply csrankings-j.csv:970]: Hi, the current official position of Jiajun Wen is associated professor of College of Computer Science and Software Engineering
 in Shenzhen University, which is showed in the screenshot that marked in red box. This screenshot is from the official website page of College of Computer Science and Software Engineering in Shenzhen University. "Senior research Scientist" is just an honored title, not his official position. 
screenshot

screenshot
[https://csse.szu.edu.cn/en/pages/user/index?id=835](url)

File: csrankings-s.csv (hunk 2 above)
Comment (zmh217) [reply csrankings-s.csv]: Sorry the last faculty member Jiajun Wen in this request  should be assistant professor but I made mistakes to spell as "associated professor" and closed the conversation so I can not make any comments. 

For faculty member Shu Yang, his official position is also an assistant professor. Below is the screenshot from the official website homepage of College of Computer Science and Software Engineering in Shenzhen University. Associated Researcher is the outdated title. 
Shu Yang
[https://csse.szu.edu.cn/en/pages/teacherTeam/index?p=1&s=Shu%20Yang](url)

File: csrankings-t.csv (hunk 3 above)
Comment (zmh217) [reply csrankings-t.csv]: I post the official website homepage link and screenshot from College of Computer Science and Software Engineering in Shenzhen University. Please check if this homepage is eligible? Thank you so much. [https://csse.szu.edu.cn/en/pages/user/index?id=1204](url)
Tao Dai

File: csrankings-x.csv (hunk 4 above)
Comment (zmh217) [reply csrankings-x.csv]: I post the official website homepage link and screenshot from College of Computer Science and Software Engineering in Shenzhen University. Please check if this homepage is eligible? Thank you so much.
[https://csse.szu.edu.cn/en/pages/user/index?id=556](url)

**Comment by emeryberger** [comment]
Please go ahead and update the home pages.

File: csrankings-j.csv (hunk 1 above)
Comment (zmh217) [reply csrankings-j.csv:970]: Sorry I made mistakes when typed the position of Jiajun Wen in the previous comment. Jiajun Wen should be assistant professor, not associated professor like showing in the screenshot above. Please have a check. Thank you!

**Comment by zmh217** [comment]
> Please go ahead and update the home pages.

Thank you for your suggestions. Homepages have been updated. Please have a check. Thank you so much and have a nice day!

**Pull request merged**